alphastats/DataSet_Preprocess.py

```diff
@@ -84,6 +85,7 @@ def init_preprocessing_info(
             PreprocessingStateKeys.DATA_COMPLETENESS_CUTOFF: 0,
             PreprocessingStateKeys.NUM_PG_REMOVED_DUE_TO_DATA_COMPLETENESS_CUTOFF: 0,
             PreprocessingStateKeys.MISSING_VALUES_REMOVED: False,
+            PreprocessingStateKeys.DROP_UNMEASURED_FEATURES: False,
```

Comment:
TODO: Add Key for 0 replacement
TODO: Make sure the info is stored in a sensible order for display in the sidebar